.appveyor.yml

```diff
@@ -0,0 +1,27 @@
+install:
+- set "CONDA_INSTALL_LOCN=C:\\Miniconda36-x64"
+- call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
+- conda update --yes --quiet conda
+- conda config --add channels conda-forge
+- conda install --yes clangdev posix m2-make pthreads-win32
+- copy %CONDA_INSTALL_LOCN%\Library\lib\pthreads.lib %CONDA_INSTALL_LOCN%\Library\lib\pthread.lib
+- copy %CONDA_INSTALL_LOCN%\Library\lib\pthreads.lib %CONDA_INSTALL_LOCN%\Library\lib\m.lib
```

Comment:
this is pretty hacky and not something anyone will want to do locally. would be better to fix in the build system.

Reply:
Fixed in https://github.com/flame/blis/pull/227/commits/a3e47eea2631c603a3f844c54d949d2025c87f4e